core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts

```diff
@@ -25,27 +27,36 @@ export class EditContentLayoutComponent {
 
     private readonly dotEditContentService = inject(DotEditContentService);
     isContentSaved = false;
-    formData$ = this.identifier
+
+    formData$: Observable<EditContentFormData> = this.identifier
         ? this.dotEditContentService.getContentById(this.identifier).pipe(
-              switchMap(({ contentType }) => {
+              switchMap(({ contentType, ...other }) => {
```

Comment:
we should change the name `other` for something more meaningful, what about `contentData`